File: solutions/pangram/Zig/pangram.zig
pub fn isPangram(str: []const u8) bool {
    var letters = [_]bool{false} ** 26;

    for (str) |c| {
        const lower = switch (c) {
            'A'...'Z' => c + 32,
            'a'...'z' => c,
            else => continue,
        };
        letters[lower - 'a'] = true;
    }

    for (letters) |seen| {
        if (!seen) return false;
    }
    return true;
}

File: solutions/pangram/Go/pangram.go
package pangram
import "strings"

func IsPangram(input string) bool {
    input = strings.ToLower(input)

    letters := make(map[rune]bool)

    for _, char := range input {
        if char >= 'a' && char <= 'z' {
            letters[char] = true
        }
    }

    return len(letters) == 26
}